src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/sentencing/SentencingAdjustmentService.kt

```diff
@@ -261,8 +261,38 @@ class SentencingAdjustmentService(
 
   @Transactional(propagation = Propagation.MANDATORY)
   fun convertAdjustmentsToRecallEquivalents(sentences: List<OffenderSentence>) {
-    sentences.forEach { sentence -> sentence.adjustments.filter { it.sentenceAdjustment.id == REMAND_CODE }.forEach { it.sentenceAdjustment = sentenceAdjustmentRepository.findByIdOrNull(RECALL_REMAND_CODE)!! } }
```

Comment:
IntelliJ was complaining about duplicate code due to the forEach/filter so extracted it out to something less readable ....